MusicNotationKit/MusicNotationKit/NoteDuration.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  NoteDuration.swift
+//  MusicNotationKit
+//
+//  Created by Kyle Sherman on 8/20/16.
+//  Copyright © 2016 Kyle Sherman. All rights reserved.
+//
+
+public struct NoteDuration {
+
+    public enum Value: CustomDebugStringConvertible {
+        case large
+        case long
+        case doubleWhole
+        case whole
+        case half
+        case quarter
+        case eighth
+        case sixteenth
+        case thirtySecond
+        case sixtyFourth
+        case oneTwentyEighth
+        case twoFiftySixth
+
+        public var debugDescription: String {
+            switch self {
+            case .large: return "8"
+            case .long: return "4"
+            case .doubleWhole: return "2"
+            case .whole: return "1"
+            case .half: return "1/2"
+            case .quarter: return "1/4"
+            case .eighth: return "1/8"
+            case .sixteenth: return "1/16"
+            case .thirtySecond: return "1/32"
+            case .sixtyFourth: return "1/64"
+            case .oneTwentyEighth: return "1/128"
+            case .twoFiftySixth: return "1/256"
+            }
+        }
+    }
+
+    public let value: Value
+    public let dotCount: Int
+    public var timeSignatureValue: Int? {
```

Comment:
Are you planning to use this in the duration calculations?  .large, .long. doubleWhole would have to be 1/8, 1/4,  and 1/2, since this returns the inverse value.  I think we have them in the enum, we should support them here.